src/main/resources/assets/carpet/scripts/distance_rc_1.sc

```diff
@@ -0,0 +1,55 @@
+import('math','_euclidean','_manhattan','_round');
+
+__config()->{
+    'commands'->{
+        'from <start_pos>'->'set_start',
+        'from <start_pos> to <end_pos>'->'_calculate',
+        'to <end_pos>'->_(end_pos)->(_calculate(null,end_pos))
+    }
+};
+
+global_positions={};
+
+set_start(pos)->(
+    global_positions:player() = pos;
```

Comment:
Since app is player scope, is there really a need to save it in a map with the player's name?

Reply:
true, I forgot that bit.